src/qml/ComposeBar.qml

```diff
@@ -605,10 +614,12 @@ Item {
         }
 
         onExpandedChanged: {
-            if (expanded && Qt.inputMethod.visible) {
-                attachmentPanel.forceActiveFocus()
+            //in landscape mode, we don't have enough place to display the attachment, lets dismiss keyboard
+            if (expanded && Qt.inputMethod.visible && (messages.landscape)) {
+                Qt.inputMethod.hide()
```

Comment:
Will this also cause the keyboard to dismiss on desktop, when the window is wider than it is tall?

Reply:
Humm, oh yes probably, that 's why there is attachmentPanel.forceActiveFocus() currently on xenial branch ?